analysis/README.md

```diff
@@ -61,7 +70,21 @@ This installs all binaries to run the analysis. Java 11 is recommended, while Ja
 
 To run it you can call `ccsh`
 
-## Installation
+## Installation via Docker
+
+You can use Codecharta analysis in multiple ways. This section will deal with how to use the analysis as a standalone container. For information on how to use with docker compose, please check out our [docker compose documentation](https://maibornwolff.github.io/codecharta/docs/docker-containers/).
+We assume that you already installed docker, if not, you have to do that before!
+
+To containerize the analysis, please follow the below listed steps.
+
+-   Navigate into the directory you want to analyse with CodeCharta. There are multiple ways to use the docker image:
+    1. Start the docker container and a bash shell in it with `docker run --name codecharta-analysis -it -v $(pwd):$(pwd) -w $(pwd) codecharta/codecharta-analysis bash`. This runs the image, names it `codecharta-analysis` and mounts the current directory and sets it as the working directory in the image. You can now use the codecharta shell or any other of the tools installed in the container via the command line.
+    2. Start the docker container and directly use some command (like the ccsh): `docker run --name codecharta-analysis -it -v $(pwd):$(pwd) -w $(pwd) codecharta/codecharta-analysis ccsh`. This starts the Ccsh without any commands, which will open an interactive codecharta shell that will guide you through the parsing/analyzing process. Your current directory will be your previous working directory.
```

Comment:
I don't understand this sentece `Your current directory will be your previous working directory.`.

Reply:
I was trying to say that the previous working directory of the terminal used to run the command, will be the working directory of the started container. I will change the sentence so this gets more clear